dev/Breadcrumb/BreadcrumbBar.cpp

```diff
@@ -450,7 +450,7 @@ bool BreadcrumbBar::MoveFocus(int indexIncrement)
         {
```

Comment:
I would exit early instead of doing a bunch of unneeded work when indexIncrement == 0:

```
    if (indexIncrement == 0)
    {
        return false;
    }
    if (auto const& itemsRepeater = m_itemsRepeater.get())
    {
        ...
```